packages/yourloops/lib/patient/patient.api.ts

```diff
@@ -64,6 +64,22 @@ export default class PatientApi {
     }
   }
 
+  static async getPatient(userId: string): Promise<Patient[]> {
+    try {
+      const patients: Patient[] = []
+      const { data } = await HttpService.get<Patient>({ url: `/bff/v1/patients/${userId}` })
+      patients.push(data)
+      return patients
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏
```suggestion
      const { data } = await HttpService.get<Patient>({ url: `/bff/v1/patients/${userId}` })
      return [data]
```